spec/lib/administrate/resource_resolver_spec.rb

```diff
@@ -61,17 +63,39 @@ module Library; class Book; end; end
   end
 
   describe "#resource_title" do
+    # rubocop:disable Rails/ApplicationRecord
     it "handles global-namepsace models" do
+      class User < ActiveRecord::Base; self.abstract_class = true; end
```

Comment:
What's stopping us from using `ApplicationRecord` here?

Reply:
It’s not in scope when just running the file. Could be required of course.